tempesta_fw/t/unit/sched_helper.c

```diff
@@ -87,36 +182,43 @@ test_create_srv(const char *in_addr, TfwSrvGroup *sg)
 		BUG_ON(r);
 	}
 
-	BUG_ON(tfw_create_server_ptr == NULL);
-	srv = tfw_create_server_ptr(&addr);
+	srv = tfw_create_server(&addr);
 	BUG_ON(!srv);
 
 	tfw_sg_add(sg, srv);
 
 	return srv;
 }
-
-TfwSrvConnection *
+static struct kmem_cache *test_conn_cache;
```

Comment:
Why do you declare the variable here? It should be at begin of the file as other variables.

Empty lines should be here.